test/functional/rpc_blockchain.py

```diff
@@ -438,6 +438,7 @@ def _test_getdifficulty(self):
     def _test_getnetworkhashps(self):
         self.log.info("Test getnetworkhashps")
         hashes_per_second = self.nodes[0].getnetworkhashps()
+        hashes_per_second_since_diff_adjustment = self.nodes[0].getnetworkhashps(-1, -1)
```

Comment:
The second argument of `getnetworkhashps` (height) has already {-1} as default so you don't need to pass it I believe.
```suggestion
        hashes_per_second_since_diff_adjustment = self.nodes[0].getnetworkhashps(-1)
```

Reply:
updated with the cherry-picked commit 17e505a